src/getHumanAge.test.js

```diff
@@ -7,4 +7,34 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return 0 if values < 15`, () => {
+    expect(getHumanAge(13, 5))
+      .toEqual([0, 0]);
+  });
+
+  test(`should return 1 if values >= 15 and < 24`, () => {
+    expect(getHumanAge(15, 22))
+      .toEqual([1, 1]);
+  });
+
+  test(`should return 2 if catAge >= 24 and < 28`, () => {
+    expect(getHumanAge(24, 0))
+      .toEqual([2, 0]);
+  });
+
+  test(`should return 2 if dogAge >= 24 and < 29`, () => {
```

Comment:
The test description is not clear. Test descriptions should be written in a way that they clearly describe what functionality is being tested.

Reply:
Done